### PR TITLE
Remove me from codeowners 😭

### PR DESCRIPTION
**Pull request by analogrelay**
😭😭😭😭😭

### CODE REVIEW

**Review by JamesNK — APPROVED** [review]
https://youtu.be/NoOhnrjdYOc?t=20

**Comment by anurse** [comment]
https://www.youtube.com/watch?v=k1jUTCH-83U#t=34s

**Review by Tratcher — APPROVED** [review]

**Review by halter73 — APPROVED** [review]
:cry:

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -10,15 +10,15 @@
...
-/src/Middleware/Rewrite/        @jkotalik @anurse
+/src/DefaultBuilder/            @tratcher
+/src/Hosting/                   @tratcher
+/src/Http/                      @tratcher @jkotalik
+/src/Middleware/                @tratcher
+/src/Middleware/HttpsPolicy/    @jkotalik
+/src/Middleware/Rewrite/        @jkotalik
 # /src/ProjectTemplates/          @ryanbrandenburg
-/src/Security/                  @tratcher @anurse
-/src/Servers/                   @tratcher @jkotalik @anurse @halter73
```
Comment (jkotalik) [inline .github/CODEOWNERS:21]: nit: revert

**Comment by anurse** [comment]
I still managed to break the build :(

**Comment by BrennanConroy** [comment]
> I still managed to break the build :(

One more time for old times sake.

**Pull request merged**